### ISSUE TITLE
Suggestion: Don't commit js files that compiled from ts files

### ISSUE THREAD

**Issue by finscn** [flyover/box2d.ts#19]
the js files compiled from ts files are as same as `dist/build files`.
In most open source projects , they won't commit `dist/build files` to git.

And they will tell users how to build/compile the project.

**Comment by flyover** [comment]
Yes, I agree.  I plan on addressing this with the multiple-output (systemjs, commonjs, ...) change.

**Comment by flyover** [comment]
I think I'm going to leave them for the time being.  It's an easy way to side-by-side the output.  It's also easy to hide them in the source tree (in VS Code).
